src/common/sns.js

```diff
@@ -13,7 +14,7 @@ export const publish = async (topicArn, message) => {
     source: config.get("serviceName"),
     specVersion: "1.0",
     type: `cloud.defra.${config.get("cdpEnvironment")}.${config.get("serviceName")}.case.stage.updated`,
-    data: message
+    data: { ...message, traceParent: getTraceId() }
```

Comment:
`traceparent` should be lowercase and appear at the root of the event and separate from data.

Appreciate there are none whatsoever but would be great to also add a test to expand coverage.